packages/@ember/-internals/glimmer/tests/integration/application/rendering-test.js

```diff
@@ -1014,7 +1014,7 @@ moduleFor(
         template: 'Hi {{this.person.name}} from component',
       });
 
-      let expectedBacktrackingMessage = /modified `Person \(Ben\)` twice in a single render\. It was first rendered as `this\.model\.name` in "template:my-app\/templates\/routeWithError\.hbs" and then modified later in "component:foo"/;
+      let expectedBacktrackingMessage = /You attempted to dirty `name` on `Person \(Ben\)`, but it had already been consumed previously in the same render/;
```

Comment:
Totally understand the value there, I tried to preserve this but in a slightly different way. The new message includes two stack traces, one from when the object was first used, and one from when the object is dirtied after being used.

Consumption:

![Screen Shot 2019-11-17 at 9 57 19 AM](https://user-images.githubusercontent.com/685518/69011577-f60ef400-0920-11ea-9dc3-9aeafb4a9017.png)

Dirtying:

![Screen Shot 2019-11-17 at 9 57 30 AM](https://user-images.githubusercontent.com/685518/69011582-02934c80-0921-11ea-970c-f1169139d462.png)

This is for one of the basic tests, so it's definitely possible that these stack traces could get long/fill up with framework code (e.g. if you use `EmberObject.get`, it'll have to go through a lot more layers), but the value is that with tracked properties, you can actually click and go to the exact line that caused the issue.

Maybe we can try to figure out a way to put the error messages closer to the actual `get`/`set` so that there aren't as many layers of framework code in between?

Reply:
Hmm, the original assertion included the source template name and I don't think any stack trace will be able to do that (IMHO making that part of this a decent regression), but I totally like the stack trace RE: where the mutation occurs. RE: the starting location of the stack, we can definitely trim the top couple of frames ([here](https://github.com/qunitjs/qunit/blob/master/src/core/stacktrace.js) is where QUnit does that).

Reply:
Added back context using the `debugRenderTree`